components/Downloads/PrimaryDownloadMatrix.tsx

```diff
@@ -60,9 +54,9 @@ const PrimaryDownloadMatrix: FC<PrimaryDownloadMatrixProps> = ({
         <ul className="no-padding download-platform">
           <li>
             <a
-              href={`https://nodejs.org/dist/${node}/node-${node}-x86.msi`}
+              href={`https://nodejs.org/dist/${release.versionWithPrefix}/node-${release.versionWithPrefix}-x86.msi`}
```

Comment:
```suggestion
              href={`/dist/${release.versionWithPrefix}/node-${release.versionWithPrefix}-x86.msi`}
```

Reply:
I guess we can remove the nodejs.org prefix from all these links

Reply:
Yeah, it will work...

However, I am +0 on this.
Why? Because when we deploy the preview or running in local machine, it will also use the preview link or localhost which IMO seems weird (we can't test the download link by actually trying it).